setup-centos-rhel-vm.sh

```diff
@@ -98,9 +98,13 @@ function serviceStatusCheck() {
       done
 }
 
+
```

Comment:
Any reason for these newlines?

Reply:
No specific reason, we can remove the new lines @aditya-lb